files/image_config/interfaces/interfaces.j2

```diff
@@ -76,7 +76,7 @@ auto {{ member }}
 allow-hotplug {{ member }}
 iface {{ member }} inet manual
     pre-up teamdctl {{ pc }} port add {{ member }} || true
-    post-up ifconfig {{ member }} up
+    post-up ip link show {{ pc }} && ifconfig {{ member }} up
```

Comment:
when we do ifdown/ifup after the interfaces become part of the LAG, we will need this.